Signal/src/ViewControllers/HomeView/Chat List/ChatListViewController.swift

```diff
@@ -496,6 +496,32 @@ public extension ChatListViewController {
         navigationController.setViewControllers(viewControllers, animated: false)
         presentFormSheet(navigationController, animated: true, completion: completion)
     }
+
+    /// Verifies that the currently selected cell matches the provided thread's uniqueId.
```

Comment:
nit: `matches the provided thread`, to avoid leaking an implementation detail (the `uniqueId`). I don't care that much about this.

Reply:
Updated!